languages/javascript/exercises/concept/nullability/.docs/introduction.md

```diff
@@ -0,0 +1,14 @@
+## Nullability
```

Comment:
This should match the casing used in the config.json file (it will be replaced with the concept name when displayed):

```suggestion
## nullability
```

Reply:
Should change the title in `about.md` as well ?

Reply:
The `about.md` file of the concept should not have a heading.

Reply:
Ok, it has been removed